src/v/cluster/members_backend.cc

```diff
@@ -167,6 +167,9 @@ ss::future<> members_backend::reconciliation_loop() {
             auto ec = co_await reconcile();
             vlog(
               clusterlog.trace, "reconciliation loop result: {}", ec.message());
+            if (ec == errc::shutting_down) {
+                co_return;
+            }
             if (!ec) {
```

Comment:
what would happen if some code accidentally/bug returned shutting_down error here? seems like the broker would have to be restarted. in other words, why not rely on the abort source signal to stop this loop

Reply:
Good point, will remove this early `co_return`.